### PR TITLE
Add KMagick (Kotlin) to bindings list

### PR DESCRIPTION
**Pull request by cherryleafroad**
Hello!

I created some [ImageMagick bindings for Kotlin](https://github.com/cherryleafroad/kmagick), and I added it to the bindings list on the website. 😀

### CODE REVIEW

File: include/develop.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
.../a> (Lisp), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#lua">Lua</a> (LuaJIT), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#neko">NMagick</a> (Neko/haXe), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#dot-net">Magick.NET</a> (.NET), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#pascal">PascalMagick</a> (Pascal),  <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#perl">PerlMagick</a> (Perl), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#php">IMagick</a> (PHP), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#python">PythonMagick</a> (Python), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#r">magick</a> (R), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#ruby">RMagick</a> (Ruby), <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#tcl">TclMagick</a> (Tcl/TK), or <a href="<?php echo $_SESSION['RelativePath']?>/../script/develop.php#wasm">WASM</a>. With a language interface, use ImageMagick to modify or create images dynamically and <var>automagically</var>.</p>
```
Comment (dlemstra) [inline include/develop.php:3]: This might be a good time to finally add some new lines here? Would you mind adding some after the `,`'s?

Comment (cherryleafroad) [reply include/develop.php:3]: Would a little extra formatting look better? E.g. like this?
```
The functionality of ImageMagick is typically utilized from the command-line or you can use the features from programs written in your favorite language.

Choose from these interfaces:
G2F (Ada),
MagickCore (C), 
KMagick (Kotlin),
... etc ...

With a language interface, use ImageMagick to modify or create images dynamically and automagically.
```

Comment (dlemstra) [reply include/develop.php:3]: Oh I did not mean to reformat it (no `</br>`) but just put all these interfaces on a new line to make it easier to see what is changed next time.

Comment (cherryleafroad) [reply include/develop.php]: Done, how's it look now?

**Comment by dlemstra** [comment]
Thanks for this PR!

**Pull request merged**